cmd/pmem-ns-init/main.go

```diff
@@ -100,6 +100,10 @@ func createNS(r *ndctl.Region, nsSize uint64, uselimit int, nsmode ndctl.Namespa
 			glog.Infof("MaxAvailableExtent in Region:%v is less than desired nsSize, limit nsSize to that", r.MaxAvailableExtent())
 			nsSize = r.MaxAvailableExtent()
 		}
+		// In typical case, NSize drops to zero at some point, avoid zero-size create request
+		if nsSize == 0 {
```

Comment:
@okartau we can move this check to bit ahead of AvailableSize checks.

Reply:
My idea here is, nsSize can be set to zero in both checks above, so checking it against zero covers both. And if nsSize is set to zero in upper check, the lower check will do nothing so it is safe

Reply:
yes true.